Alicerce/Sources/Logging/Log+ConsoleProvider.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Log+ConsoleProvider.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class ConsoleProvider: LogProvider {
+
+        public enum ConsoleOutput {
+            case print
+            case nslog
+        }
+
+        public var minLevel: Log.Level = .error
+        public var formatter: LogItemFormatter = Log.ItemStringFormatter()
```

Comment:
I would like to have an agreement on this, a guideline. use inference or not, i would prefer to use it in `struct`/`class` properties, otherwise it gets a little bit messy.